docs/USAGE.md

```diff
@@ -71,9 +71,16 @@ The exporter is used to output the telemetry.
 | `OTEL_EXPORTER_OTLP_ENDPOINT` | Target endpoint for OTLP exporter. More details [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md). | `http://localhost:4318` |
 | `OTEL_EXPORTER_OTLP_HEADERS` | Key-value pairs to be used as headers associated with gRPC or HTTP requests. More details [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md). | |
 | `OTEL_EXPORTER_OTLP_TIMEOUT` | Maximum time the OTLP exporter will wait for each batch export. | `1000` (ms) |
-| `OTEL_EXPORTER_OTLP_PROTOCOL` | The transport protocol. Supported values: `grpc`, `http/protobuf`. | `grpc` |
+| `OTEL_EXPORTER_OTLP_PROTOCOL` | The OTLP expoter transport protocol. Supported values: `grpc`, `http/protobuf`. [1] | `grpc` |
```

Comment:
#351 changes the default - @Kielek we need to ensure that we don't lose the link to the note when doing the merge.

Reply:
Feel free to merge, I will resolve the conflicts tommorow :)